tiatoolbox/models/dataset/abc.py

```diff
@@ -14,8 +14,8 @@ class PatchDatasetABC(ABC, torch.utils.data.Dataset):
     Attributes:
         preproc_func:
             Preprocessing function used to transform the input data. If
-            supplied, then torch.Compose will be used on the input
-            preprocs. preprocs is a list of torchvision transforms for
+            supplied, then torch. Compose will be used on the input
+            preproc. preproc is a list of torchvision or custom transforms for
```

Comment:
no spacing for funcname, while you are at it, nest them between between `.

Reply:
Do you mean to change `torch. Compose` to `torch.Compose`?

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/366/commits/d6cbda02096c839860117a2aefd43b1ebee22760